lib/rules/echidna/deliverer-change.js

```diff
@@ -0,0 +1,57 @@
+import w3cApi from 'node-w3capi';
+
+w3cApi.apiKey = process.env.W3C_API_KEY;
+
+const self = {
+    name: 'echidna.deliverer-change',
+    section: 'metadata',
+    rule: 'delivererID',
+};
+
+export const { name } = self;
+
+async function getPreviousIDs(shortname, previousUrl) {
```

Comment:
To avoid confusion, I suggest to rename that function. Also, `async` doesn't seem necessary here:
```suggestion
function getPreviousDelivererIDs(shortname, previousUrl) {
```